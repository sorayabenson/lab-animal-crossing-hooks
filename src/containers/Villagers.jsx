import React from 'react';
// import Header from '../components/Header';
// import Spinner from '../components/Spinner';
// import VillagerList from '../components/villagers/VillagerList';

const Villager = () => {
    return (<p>hello villagers</p>)
};

export default Villager;