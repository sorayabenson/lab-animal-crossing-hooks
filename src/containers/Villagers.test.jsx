import React from 'react';
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import Villagers from './Villagers';

describe('Villager component', () => {
    it('renders Villagers', () => {
        render(<Villagers />);
        
    })
})