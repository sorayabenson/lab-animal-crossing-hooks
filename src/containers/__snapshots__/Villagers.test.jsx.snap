// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Villager container renders Villagers 1`] = `
Array [
  <ul
    aria-label="villagers"
  >
    <li
      aria-label="villager"
    >
      <a
        class="villagerItem"
        href="/1"
      >
        <img
          alt="Cyrano"
          src="https://acnhapi.com/v1/images/villagers/1"
        />
        <h3
          aria-level="3"
        >
          Cyrano
        </h3>
        <p
          aria-label="villager catchphrase"
        >
          ah-CHOO
        </p>
      </a>
    </li>
    <li
      aria-label="villager"
    >
      <a
        class="villagerItem"
        href="/2"
      >
        <img
          alt="Antonio"
          src="https://acnhapi.com/v1/images/villagers/2"
        />
        <h3
          aria-level="3"
        >
          Antonio
        </h3>
        <p
          aria-label="villager catchphrase"
        >
          honk
        </p>
      </a>
    </li>
    <li
      aria-label="villager"
    >
      <a
        class="villagerItem"
        href="/3"
      >
        <img
          alt="Pango"
          src="https://acnhapi.com/v1/images/villagers/3"
        />
        <h3
          aria-level="3"
        >
          Pango
        </h3>
        <p
          aria-label="villager catchphrase"
        >
          snooooof
        </p>
      </a>
    </li>
    <li
      aria-label="villager"
    >
      <a
        class="villagerItem"
        href="/4"
      >
        <img
          alt="Anabelle"
          src="https://acnhapi.com/v1/images/villagers/4"
        />
        <h3
          aria-level="3"
        >
          Anabelle
        </h3>
        <p
          aria-label="villager catchphrase"
        >
          snorty
        </p>
      </a>
    </li>
    <li
      aria-label="villager"
    >
      <a
        class="villagerItem"
        href="/5"
      >
        <img
          alt="Snooty"
          src="https://acnhapi.com/v1/images/villagers/5"
        />
        <h3
          aria-level="3"
        >
          Snooty
        </h3>
        <p
          aria-label="villager catchphrase"
        >
          sniffff
        </p>
      </a>
    </li>
    <li
      aria-label="villager"
    >
      <a
        class="villagerItem"
        href="/6"
      >
        <img
          alt="Annalisa"
          src="https://acnhapi.com/v1/images/villagers/6"
        />
        <h3
          aria-level="3"
        >
          Annalisa
        </h3>
        <p
          aria-label="villager catchphrase"
        >
          gumdrop
        </p>
      </a>
    </li>
    <li
      aria-label="villager"
    >
      <a
        class="villagerItem"
        href="/7"
      >
        <img
          alt="Olaf"
          src="https://acnhapi.com/v1/images/villagers/7"
        />
        <h3
          aria-level="3"
        >
          Olaf
        </h3>
        <p
          aria-label="villager catchphrase"
        >
          whiffa
        </p>
      </a>
    </li>
  </ul>,
]
`;

exports[`Villager container renders Villagers 2`] = `
Array [
  <li
    aria-label="villager"
  >
    <a
      class="villagerItem"
      href="/1"
    >
      <img
        alt="Cyrano"
        src="https://acnhapi.com/v1/images/villagers/1"
      />
      <h3
        aria-level="3"
      >
        Cyrano
      </h3>
      <p
        aria-label="villager catchphrase"
      >
        ah-CHOO
      </p>
    </a>
  </li>,
  <li
    aria-label="villager"
  >
    <a
      class="villagerItem"
      href="/2"
    >
      <img
        alt="Antonio"
        src="https://acnhapi.com/v1/images/villagers/2"
      />
      <h3
        aria-level="3"
      >
        Antonio
      </h3>
      <p
        aria-label="villager catchphrase"
      >
        honk
      </p>
    </a>
  </li>,
  <li
    aria-label="villager"
  >
    <a
      class="villagerItem"
      href="/3"
    >
      <img
        alt="Pango"
        src="https://acnhapi.com/v1/images/villagers/3"
      />
      <h3
        aria-level="3"
      >
        Pango
      </h3>
      <p
        aria-label="villager catchphrase"
      >
        snooooof
      </p>
    </a>
  </li>,
  <li
    aria-label="villager"
  >
    <a
      class="villagerItem"
      href="/4"
    >
      <img
        alt="Anabelle"
        src="https://acnhapi.com/v1/images/villagers/4"
      />
      <h3
        aria-level="3"
      >
        Anabelle
      </h3>
      <p
        aria-label="villager catchphrase"
      >
        snorty
      </p>
    </a>
  </li>,
  <li
    aria-label="villager"
  >
    <a
      class="villagerItem"
      href="/5"
    >
      <img
        alt="Snooty"
        src="https://acnhapi.com/v1/images/villagers/5"
      />
      <h3
        aria-level="3"
      >
        Snooty
      </h3>
      <p
        aria-label="villager catchphrase"
      >
        sniffff
      </p>
    </a>
  </li>,
  <li
    aria-label="villager"
  >
    <a
      class="villagerItem"
      href="/6"
    >
      <img
        alt="Annalisa"
        src="https://acnhapi.com/v1/images/villagers/6"
      />
      <h3
        aria-level="3"
      >
        Annalisa
      </h3>
      <p
        aria-label="villager catchphrase"
      >
        gumdrop
      </p>
    </a>
  </li>,
  <li
    aria-label="villager"
  >
    <a
      class="villagerItem"
      href="/7"
    >
      <img
        alt="Olaf"
        src="https://acnhapi.com/v1/images/villagers/7"
      />
      <h3
        aria-level="3"
      >
        Olaf
      </h3>
      <p
        aria-label="villager catchphrase"
      >
        whiffa
      </p>
    </a>
  </li>,
]
`;
